miscellaneous/validator-service/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
No need of this file. let's gitignore it and remove from here

Reply:
Ack